poc_iot_injector/src/main.rs

```diff
@@ -11,29 +12,42 @@ pub enum Cmd {
     Server(server::Cmd),
 }
 
+impl Cmd {
+    pub async fn run(self, settings: Settings) -> Result {
+        match self {
+            Self::Generate(cmd) => cmd.run(&settings).await,
+            Self::Server(cmd) => cmd.run(&settings).await,
+        }
+    }
+}
+
 #[derive(Debug, clap::Parser)]
 #[clap(version = env!("CARGO_PKG_VERSION"))]
 #[clap(about = "Helium POC IOT Injector Server")]
```

Comment:
This value should appear in Cargo.toml for one place to maintain.
```suggestion
#[clap(about = env!("CARGO_PKG_DESCRIPTION"))]
```
See https://doc.rust-lang.org/cargo/reference/environment-variables.html